### PR TITLE
[Noetic] add missing installation steps and dependencies in CMakeLists

### PR DESCRIPTION
**Pull request by mikaelarguedas**
This also fixes some typos and linting issues reported by catkin_lint

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Thank you!  CC: @saching13

**Comment by saching13** [comment]
Thanks for the PR. and adding package install.
Wasn't aware of `catkin_lint`

File: depthai_bridge/CMakeLists.txt [hunk 1]
```diff
@@ -87,7 +87,7 @@ endif()
     catkin_package(
       INCLUDE_DIRS include
       LIBRARIES ${PROJECT_NAME}
-      CATKIN_DEPENDS depthai_ros_msgs camera_info_manager roscpp sensor_msgs std_msgs vision_msgs 
+      CATKIN_DEPENDS camera_info_manager depthai_ros_msgs roscpp sensor_msgs std_msgs stereo_msgs vision_msgs
```
Comment (saching13) [inline depthai_bridge/CMakeLists.txt:90]: https://github.com/luxonis/depthai-ros-examples/pull/7/files
In this PR it moved them to each individual line but not here. is it some bug with `catkin_lint` ?

Comment (mikaelarguedas) [reply depthai_bridge/CMakeLists.txt:90]: catkin lint only report violations, not fix them, it was easier for me to fix them by putting the dependencies on individual lines but I didn't necessarily did it consistently over all the places I modified if it was easy to rearrange the dependencies without modifying the line count

Comment (saching13) [reply depthai_bridge/CMakeLists.txt:90]: got it. Thanks.

**Pull request merged**